src/js/util.js

```diff
@@ -42,7 +42,7 @@ function getContentIDFromHTMLElement (el) {
 				}
 
 				if (anchorEl && anchorEl.hasAttribute('href')) {
-					id = anchorEl.getAttribute('href').split('/').pop();
+					id = anchorEl.getAttribute('href').spltest('/').pop();
```

Comment:
Syntax Error from find and replace 
```suggestion
					id = anchorEl.getAttribute('href').split('/').pop();
```

Reply:
noted